container-scan/action.yaml

```diff
@@ -0,0 +1,92 @@
+name: "Security Scan Composite Action"
+description: "GitHub Action for scanning container image for vulnerabilities using Lacework"
+
+inputs:
+  #  docker-config-file:
+  #    required: true
+  #    description: Path to the docker config file
+  dockerhub-user:
+    required: true
+    description: DockerHub username
+  dockerhub-password:
+    required: true
+    description: DockerHub password
+  github-token:
+    required: true
+    description: GitHub token
+  artifactory-username:
+    required: true
+    description: Artifactory username
+  artifactory-auth-token:
+    required: true
+    description: Artifactory auth token
+  lw-account-name:
+    required: true
+    description: Lacework account name
+  lw-access-token:
+    required: true
+    description: Lacework access token
+  image-name:
+    required: true
+    description: Docker image name
+
+outputs:
+  comment-id:
+    description: Comment ID of the test report
+    value: ${{ steps.comment-pr.outputs.comment-id }}
+
+runs:
+  using: composite
+  steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: Scan container image for vulnerabilities using Lacework
+      uses: lacework/lw-scanner-action@v1.4.0
+      with:
+        LW_ACCOUNT_NAME: ${{ inputs.lw-account-name }}
+        LW_ACCESS_TOKEN: ${{ inputs.lw-access-token }}
+        IMAGE_NAME: ${{ inputs.image-name }}
+        IMAGE_TAG: ${{ github.run_id }}-${{ github.run_attempt }}
```

Comment:
Should the tag be a parameter to the action? To not assume anything about the naming system the user uses on their build?